gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -91,17 +94,20 @@ public Bundle mapToBundle(RCMRIN030000UK06Message xmlMessage) {
         var referralRequests = mapReferralRequests(ehrFolder, patient);
         addEntries(bundle, referralRequests);
 
-        var observations = mapObservations(ehrExtract, patient, encounters);
+        var medicationResources = medicationRequestMapper.mapResources(ehrExtract, encounters, patient);
+        addEntries(bundle, medicationResources);
+
+        var observations = mapObservations(ehrExtract, patient, List.of()); //TODO: Provide list of encounters
         addEntries(bundle, observations);
 
-        var immunizations = mapImmunizations(ehrExtract, patient, encounters);
+        var immunizations = mapImmunizations(ehrExtract, patient, List.of()); //TODO: Provide list of encounters (NIAD-1961)
```

Comment:
It seems like Rory's changes in this file were discarded :(

Reply:
Added the encounter list to each method, will have to update the INT test file to accomodate this